build-logic/convention/src/main/kotlin/fr/androidmakers/plugin/SigningConfigPlugin.kt

```diff
@@ -38,7 +38,7 @@ class SigningConfigPlugin : Plugin<Project> {
             isMinifyEnabled = true
             isShrinkResources = true
             proguardFiles(
-                getDefaultProguardFile("proguard-android.txt"),
```

Comment:
Wouldn't it be better to use proguard-android-optimized here to benefit from default rules with all possible optimizations?

Reply:
These rules are better but still way too conservative in my opinion, and they haven't been updated for ages. I can give you a detailed explanation later.

Reply:
I am aware that they are outdated. My thought was it might be still safer for a small project.

I totally agree that adding rules based on your project is better.